Turkish December 2016 Update

elifkus opened the pull request:
Several fixes for Turkish Language, syncing with CrowdIn.

elifkus commented:
I tested the code on Gitbook. It seems to work fine. 

https://elifkus.gitbooks.io/django-girls-turkce-preview/content/

tricoder42 commented on tr/SUMMARY.md:
```diff
@@ -1,26 +1,51 @@
-* [Uygulamanı genişlet](extend_your_application/README.md)
-* [Django formları](django_forms/README.md)
-* [Sırada ne var?](whats_next/README.md)
+*   [Giriş][1]
```
Instead of:

```markdown
[Giriş][1]

[1]: README.md
```

it should be:

```markdown
[Giriş](README.md)
```

You have to do it manually, because crowding generates this file in incorrect format :/

tricoder42 commented on tr/css/README.md:
````diff
@@ -10,18 +10,19 @@ Fakat tekrar en baştan başlamak istemiyoruz, değil mi? Bir kez daha, programc
-<link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css">
-<link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap-theme.min.css">
-```
+‘’’html
````
I think these are incorrect "backticks". Should be ``` instead of ‘’’

tricoder42 commented on tr/deploy/README.md:
````diff
@@ -30,60 +30,61 @@ Git, kod reposu (veya "repository") denen belli dosyaların değişikliklerini i
 $ git config --global user.name "Adınız" 
 $ git config --global user.email you@example.com
-```
+‘’’    
````
Incorrect backticks (should be ```)

tricoder42 commented on tr/deploy/README.md:
````diff
@@ -30,60 +30,61 @@ Git, kod reposu (veya "repository") denen belli dosyaların değişikliklerini i
 
-```
+
+‘’’    
````
Also here, incorrect backticks (should be ```).

Maybe just search for all occurences of ‘’’ and replace them with ``` ;)

elifkus commented:
Thank you for reviewing. 

I fixed all of them. (I hope.) 

I actually thought that I replaced all of the wrong quotes with grep. But apparently some of them managed to escape.

tricoder42 commented:
Looks awesome! Let's merge it and see if it builds 🎉 👍
